lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1416,13 +1416,16 @@ static llvm::Optional<StringRef> GetDSYMBundle(Module &module) {
 }
 
 static std::string GetSDKPath(std::string m_description, XcodeSDK sdk) {
-  auto sdk_path_or_err = HostInfo::GetXcodeSDKPath(sdk);
+  llvm::Expected<llvm::StringRef> sdk_path_or_err = HostInfo::GetSDKPath();
```

Comment:
If we renamed GetXcodeSDKPath to GetSDKPath upstream, would that simplify your patch here?

Reply:
We could do that, but the extra parameters would also make this odd.  The other platforms do not have a `XcodeSDK`.  The new entry point is actually very easily portable to other platforms.

Reply:
If we make it an optional I don't see why it would be odd. I agree with Adrian that this should probably go upstream.

Reply:
Done.